experiment/run_experiment.py

```diff
@@ -95,11 +110,24 @@ def read_and_validate_experiment_config(config_filename: str) -> Dict:
                 param, str(value))
             continue
 
-        if param in bucket_params and not value.startswith('gs://'):
-            valid = False
-            logs.error(
-                'Config parameter "%s" is "%s". It must start with gs://.',
-                param, value)
+        if param in filestore_params:
+            # Checks file storage paths format respectively.
+            if not config.get('local_experiment', False):
+                if not value.startswith('gs://'):
+                    valid = False
+                    logs.error(
+                        'Config parameter "%s" is "%s".' \
```

Comment:
I generally avoid backslashes.
They aren't needed in this case, Python will join the strings.